.travis.yml

```diff
@@ -5,6 +5,9 @@ node_js:
   - '0.11'
   - 'iojs-v1.0'
   - 'iojs-v1.1'
+  - '0.12'
+  - '4.1'
+  - '5.0'
```

Comment:
ah, wasn't aware, I was just following the conventions [here](http://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Provided-Node.js-Versions), which always seemed to have the minor provided.